### PR TITLE
sig-cluster-lifecycle: update the meeting time to 8AM PT

### PR DESCRIPTION
**Pull request by neolit123**
we moved the meeting one hour earlier.
the change is already reflected in the meeting notes doc and community calendar.

also added slack for image-builder as part of this PR.

/sig cluster-lifecycle
/kind documentation

### CODE REVIEW

**Comment by fabriziopandini** [comment]
+1

**Comment by mrbobbytables** [comment]
@neolit123 if you could rebase and rerun make, I can approve quick 👍

File: sig-cluster-lifecycle/README.md [hunk 1]
```diff
@@ -135,6 +135,8 @@ The following [subprojects][subproject-definition] are owned by sig-cluster-life
 ### image-builder
 - **Owners:**
   - https://raw.githubusercontent.com/kubernetes-sigs/image-builder/master/OWNERS
+- **Contact:**
+  - Slack: [#image-builder](https://kubernetes.slack.com/messages/image-builder)
```
Comment (neolit123) [inline sig-cluster-lifecycle/README.md:139]: @mrbobbytables rebased.
also added slack for image-builder as part of this PR

**Comment by mrbobbytables** [comment]
/lgtm
/approve
applying both as it was previously lgtm'ed, and don't want further conflicts 👍

**Pull request merged**